src/platforms/python/gcp_functions.mdx

```diff
@@ -17,24 +18,50 @@ You can use the GCP Functions integration for the Python SDK like this:
 
 ```python
 import sentry_sdk
-from sentry_sdk.integrations.serverless import serverless_function
+from sentry_sdk.integrations.gcp import GcpIntegration
 
-sentry_sdk.init(dsn="___PUBLIC_DSN___")
+sentry_sdk.init(
+    dsn="___PUBLIC_DSN___",
+    integrations=GcpIntegration()]
+)
 
-@serverless_function
-def my_function(...): ...
+def my_function(event, context):
+    ...
 ```
 
-Use the generic integration by calling the `serverless_function` decorator. Decorators wrap a function and modify its behavior. Then, deploy and test the function. Checkout Sentry's [sample apps](https://github.com/getsentry/examples) for detailed examples.
+Checkout Sentry's [GCP sample apps](https://github.com/getsentry/examples/tree/master/gcp-cloud-functions) for detailed examples.
+
+### Enable Timeout Warning
+Update the sentry initialization to set ```timeout_warning``` to ```true```
+```python
+sentry_sdk.init(
+    dsn="___PUBLIC_DSN___",
+    integrations=[GcpIntegration(timeout_warning=True)]
+)
+```
+The timeout warning is sent only if the "timeout" in the GCP Funtion configuration is set to a value greater than one second.
+
+ 
+<Alert level="info" title="Note"><markdown>
```

Comment:
This alert box does not make sense in "Enable Timeout Warning", move before line 34